datasafe-encryption/datasafe-encryption-api/pom.xml

```diff
@@ -14,12 +14,10 @@
             <artifactId>datasafe-types-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-		<dependency>
-			<groupId>de.adorsys.keymanagement</groupId>
-			<artifactId>juggler-bouncycastle</artifactId>
-		</dependency>
-
-
+        <dependency>
+            <groupId>de.adorsys.keymanagement</groupId>
```

Comment:
You should use de.adorsys.keymanagement:api in API
And in impl use de.adorsys.keymanagement:juggler-bouncycastle